Practice/style.css

```diff
@@ -0,0 +1,77 @@
+body{
+    background-color: gray;
+}
+
+div{
+    font-size: 26px;
+}
+
+div{
```

Comment:
de regula, in acelasi fisier nu se foloseste de 2 ori acelasi selector
Aici, corect era un singur div cu
div {  
    font-size: 26px;
    font-style: oblique;
    font-weight: 16px;
}